### ISSUE TITLE
Confusing use of "Sensitive Data"

### ISSUE THREAD

**Issue by woodbe** [commoncriteria/application#192]
FDP_DAR_EXT.1 is for sensitive application data, but then the third bullet states that sensitive data could be protected according to FCS_STO_EXT.1, which is specifically about credentials. While this does meet the definition of sensitive data (which includes credentials and keys), it seems to be putting too many things into that bucket.

I think it would be best to keep credentials and keys as one item (covered specifically by FCS_STO_EXT.1) and then Sensitive data to be everything else.

Just to keep things clean, having "data" and "keys" mixed as "sensitive data" is confusing, and ideally should be minimized.

**Comment by jmcdaniels** [comment]
I think that is how it originally was, but we had to add the STO reference in DAR otherwise it looked like you were mandated to also encrypt credentials.

**Comment by jvdsn** [comment]
I do think there is value in allowing FCS_STO_EXT.1 here. Otherwise the SFR would need to be written to explicitly say "sensitive data (excluding credentials)" everywhere, which is probably infeasible at this point.

**Comment by woodbe** [comment]
If there was a new definition for Sensitive Data that excluded credentials it wouldn't need to be changed here.

Change the Technical Term definition:

>Sensitive data may include all user or enterprise data or may be specific application data such as emails, messaging, documents, calendar items, and contacts. Sensitive data must minimally include PII, credentials, and keys. Sensitive data shall be identified in the application’s TSS by the ST author.

to not have the middle sentence of "Sensitive data must minimally include PII, credentials, and keys." 

The PII Technical Term definition would then say something to the effect of (in addition to what is there) "this is a subset of Sensitive Data".

It would mean checking where Sensitive Data is used elsewhere to make sure that keys/credentials are used instead of Sensitive Data, but it would be cleaner.

**Comment by jvdsn** [comment]
> If there was a new definition for Sensitive Data that excluded credentials it wouldn't need to be changed here.

Right, but perhaps this impacts other areas? This is where I could find sensitive data used in the PP (outside FDP_DAR_EXT.1):

> T.PHYSICAL_ACCESS: An attacker may try to access sensitive data at rest.

Would have to be replaced by "sensitive data or credentials at rest"

> FDP_DEC_EXT.1.2: Application Note: "Sensitive information repositories" are defined as
those collections of sensitive data that could be expected to be shared
among some applications, users, or user roles, but to which not all of
these would ordinarily require access.

Maybe doesn't have to be updated?

> FMT_MEC_EXT.1.1: Application Note: Configuration options that are stored remotely are
not subject to this requirement. Sensitive Data is generally not
considered part of configuration options and should be stored according
to FDP_DAR_EXT.1 or FCS_STO_EXT.1.

Would be replaced by "Sensitive data or credentials are generally not ..."

> FTP_DIT_EXT.1

Probably also would have to be updated to disallow sending plaintext credentials over the network.

**Comment by jmcdaniels** [comment]
For the moment I am landing on separating them not being worth the effort to split them apart seeing as this STO fix is already in place.  I am going to leave this issue open for future revision consideration, if we get TRRTs in the future indicating this is causing issues or confusion it can be addressed then.

**Comment by woodbe** [comment]
Just commenting that this works for me, pushing it out to discuss is good.
